articles/relnotes/index.md

```diff
@@ -27,7 +27,7 @@ This release adds new Q# language syntax:
 * Add a [shorthand way to express specializations of quantum operations](xref:microsoft.quantum.language.type-model#functors) (control and adjoints) with `+` operators.  The old syntax is deprecated.  Programs that use the old syntax (e.g., `: adjoint`) will continue to work, but a compile time warning will be generated.  
 * Add a new operator for [copy-and-update](xref:microsoft.quantum.language.expressions#copy-and-update-expressions), `w/`, can be used to express array creation as a modification of an existing array.
 * Add the common [apply-and-upate statement](xref:microsoft.quantum.qsharp-ref.statements#rebinding-of-mutable-symbols), e.g., `+=`, `w/=`.
-* Add a way to specify a short name for namespaces in [open directives](xref:microsoft.quantum.language.file-structure#open-directives).
+* Add a way to specify a short name for namespaces in [Open Directives](xref:microsoft.quantum.language.file-structure#open-directives).
```

Comment:
I don't think it makes sense to capitalize here - we don't capitalize anything else; set-statements, if-statements, namespace declaration … - why capitalize this one?

Reply:
yes